### ISSUE TITLE
Adopt NO_COLOR standard

### ISSUE THREAD

**Issue by ilai-deutel** [ilai-deutel/kibi#465]
Standard: https://no-color.org/

For Kibi:

- [ ] Disable ANSI color escape codes when `NO_COLOR` environment variable is present and not an empty string.

- [ ] [Optional] Add color parameter in configuration
    > User-level configuration files and per-instance command-line arguments should override the NO_COLOR environment variable. A user should be able to export NO_COLOR=1 in their shell configuration file as a default, but configure a specific program in its configuration file to specifically enable color.
   
    This is optional, but potentially nice-to-have. Otherwise users can always run `NO_COLOR=1 kibi` or `NO_COLOR= kibi`, which is not as ergonomic.
